cpp/include/ucxx/memory_handle.h

```diff
@@ -28,6 +28,7 @@ class MemoryHandle : public Component {
   ucp_mem_h _handle{};       ///< The UCP handle to the memory allocation.
   size_t _size{0};           ///< The actual allocation size.
   uint64_t _baseAddress{0};  ///< The allocation's base address.
+  ucs_memory_type_t _memoryType{UCS_MEMORY_TYPE_HOST};
```

Comment:
Maybe add a comment, now that the other three have one :)

Reply:
Nice catch, done via https://github.com/rapidsai/ucxx/pull/246/commits/a0c0319937780af7dcbb76830f22ea048bb58d5f .